Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/BaseObjectCollection.cs

```diff
@@ -1,41 +1,28 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
 using System;
 using System.Collections.Generic;
 using UnityEngine;
-using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
 
 namespace Microsoft.MixedReality.Toolkit.SDK.UX.Collections
 {
     public abstract class BaseObjectCollection : MonoBehaviour
     {
-        #region public members
-        protected Action<BaseObjectCollection> onCollectionUpdated;
-
         /// <summary>
         /// Action called when collection is updated
         /// </summary>
-        public Action<BaseObjectCollection> OnCollectionUpdated
-        {
-            get { return onCollectionUpdated; }
-            set { onCollectionUpdated = value; }
-        }
-
-        protected List<ObjectCollectionNode> nodeList = new List<ObjectCollectionNode>();
+        public Action<BaseObjectCollection> OnCollectionUpdated { get; set; }
 
         /// <summary>
         /// List of objects with generated data on the object.
         /// </summary>
-        public List<ObjectCollectionNode> NodeList
-        {
-            get { return nodeList; }
-            set { nodeList = value; }
-        }
+        protected List<ObjectCollectionNode> NodeList { get; } = new List<ObjectCollectionNode>();
 
         [Tooltip("Whether to include space for inactive transforms in the layout")]
         [SerializeField]
-        protected bool ignoreInactiveTransforms = true;
```

Comment:
Why are you changing the visibility of these? Classes may need to access them.

Reply:
I'm not. The property accessor is public.

Reply:
Besides it's always easier to give access than to take it away.